backend/app/api/endpoints/openapi_responses.py

```
@@ -321,9 +328,288 @@ async def create_response(
+        finally:
+            query_db.close()
+
+    # Check if SSE mode is supported
+    supports_sse = execution_dispatcher.supports_streaming(execution_request)
+
+    # Non-SSE mode: dispatch and return queued response
+    if not supports_sse:
+        _close_db()
+        asyncio.create_task(
+            execution_dispatcher.dispatch(execution_request, emitter=None)
+        )
```

Ruff (0.15.5)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)